source/socket-message.js

```diff
@@ -243,17 +237,8 @@ Skylink.prototype.startRecording = function (callback) {
  *   <li>If recording session has been stopped successfully: <ol>
  *   <li><a href="#event_recordingState"><code>recordingState</code> event</a>
  *   triggers parameter payload <code>state</code> as <code>START</code>.
```

Comment:
should be changed to: 
* triggers parameter pay load <code>state</code> as <code>STOP</code>.